vendor/parse-wyc/lib/dune

```diff
@@ -1,6 +1,6 @@
 (library
  (name parse_wyc)
- (libraries menhirLib ocaml-migrate-parsetree ppxlib)
+ (libraries menhirLib ppxlib)
```

Comment:
we still directly depend on OMP in parse_wyc ..

Reply:
oh indeed